js/player.js

```diff
@@ -14,15 +14,21 @@ import * as FrameURL from "./player/FrameURL";
 
 window.addEventListener("load", function () {
 
-    Presentation.init(document.querySelector("svg"));
-    Viewport.init(Presentation).onLoad();
+    var presentation = Presentation.init(document.querySelector("svg"));
+    var viewport = Viewport.init(Presentation).onLoad();
```

Comment:
Actually, `Presentation.init()` and `Viewport.init()` return the original `Presentation` and `Viewport` objects. There is no need to store them to local variables.